### PR TITLE
Update openapi extractor

### PR DESCRIPTION
**Pull request by provokateurin**
## Summary

Updated to the latest version and applied some required fixes.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/files/appinfo/routes.php [hunk 1]
```diff
@@ -93,11 +93,6 @@
 				'url' => '/api/v1/views',
 				'verb' => 'GET'
 			],
-			[
-				'name' => 'Api#getViewConfig',
```
Comment (nickvergessen) [inline apps/files/appinfo/routes.php:97]: 🤔

File: vendor-bin/openapi-extractor/composer.json [hunk 2]
```diff
@@ -11,6 +11,6 @@
         }
     },
 	"require": {
-		"nextcloud/openapi-extractor": "dev-main#c0c80648c1d7ede03537460dfbc472ff7146f294"
+		"nextcloud/openapi-extractor": "dev-main#05efabc561dbf253d251b3c2455e0857aa0ce577"
```
Comment (nickvergessen) [inline vendor-bin/openapi-extractor/composer.json:14]:
```suggestion
		"nextcloud/openapi-extractor": "dev-main"
```

File: apps/settings/appinfo/routes.php [hunk 3]
```diff
@@ -31,9 +31,7 @@
 		'AuthSettings' => ['url' => '/settings/personal/authtokens' , 'root' => ''],
 	],
 	'routes' => [
-		['name' => 'AuthorizedGroup#create', 'url' => '/settings/authorizedgroups', 'verb' => 'POST'],
```
Comment (nickvergessen) [inline apps/settings/appinfo/routes.php:34]: Also here, you remove routes?

File: apps/files/appinfo/routes.php (hunk 1 above)
Comment (provokateurin) [reply apps/files/appinfo/routes.php:97]: This method does not exist in the controller and thus an error was thrown

File: apps/settings/appinfo/routes.php (hunk 3 above)
Comment (provokateurin) [reply apps/settings/appinfo/routes.php:34]: Same as above, the method does not exist in the controller

**Review by nickvergessen — APPROVED** [review]

**Review by andrey18106 — APPROVED** [review]

**Review by bigcat88 — APPROVED** [review]

**Comment by provokateurin** [comment]
I have one more change in openapi-extractor that I would like to include here instead of creating yet another PR.

**Pull request merged**